modules/gitops-release-notes-1-14-0.adoc

```diff
@@ -105,4 +131,14 @@ data:
 <1> Specifies the name of the ApplicationSet GitLab SCM Provider.
 <2> Specifies the namespace where the ApplicationSet GitLab SCM Provider is defined.
 +
-For more information, see link:https://argocd-operator.readthedocs.io/en/latest/reference/argocd/#add-self-signed-tls-certificate-for-gitlab-scm-provider-to-applicationsets-controller[Add Self-signed TLS Certificate for Gitlab SCM Provider to ApplicationSet Controller].
\ No newline at end of file
+For more information, see link:https://argocd-operator.readthedocs.io/en/latest/reference/argocd/#add-self-signed-tls-certificate-for-gitlab-scm-provider-to-applicationsets-controller[Add Self-signed TLS Certificate for Gitlab SCM Provider to ApplicationSet Controller].
+
+[id="breaking-change-1-14-0_{context}"]
+== Breaking change
+
+[id="cluster-secrets-with-empty-values_{context}"]
+=== Cluster secrets with empty secret values
+
+* When you upgrade to {gitops-title} v1.14, cluster secrets with an empty project value will no longer be classified as global secrets applicable to all `Applications` and `ApplicationSets`. In previous versions of {gitops-shortname}, any `Application` or `ApplicationSet` could use a cluster secret that corresponds to the URL specified in the `repoUrl` field. Starting from {gitops-title} v1.14, only cluster secrets that are scoped to an application's project will be considered for matching a repository URL. As a result, if a cluster secret is designated for example to project A, an application assigned to project B will be unable to access that secret. To enable the use of applications across various projects for cluster secrets, you must deselect the project field. link:https://issues.redhat.com/browse/GITOPS-5623[GITOPS-5623]
```

Comment:
```suggestion
* When you upgrade to {gitops-title} v1.14, cluster secrets with an empty project value will no longer be classified as global secrets applicable to all `Applications` and `ApplicationSets`. In previous versions of {gitops-shortname}, any `Application` or `ApplicationSet` could use a cluster secret that corresponds to the URL specified in the `repoUrl` field. Starting from {gitops-title} v1.14, only cluster secrets that are scoped to an application's project will be considered for matching a repository URL. As a result, if a cluster secret is designated for example to project A, an application assigned to project B will be unable to access that secret. To enable the use of applications across various projects for cluster secrets, you must deselect the project field. link:https://issues.redhat.com/browse/GITOPS-5623[GITOPS-5623]
```

@anandf and @Dhruv-Soni11 what do we mean when we say unset the  `project` field here? You have used deselect here. Where is the action happening? YAML file? web console UI or the argo CD UI?

Reply:
Hi @Srivaralakshmi - I have had a detailed discussion with @anandf and due to the unavailability of a cluster, he explained the scenario on call. Basically, in this scenario, we have to remove this field from the YAML as per his information.